enos/enos-scenario-smoke.hcl

```diff
@@ -167,35 +183,80 @@ scenario "smoke" {
   }
 
   step "verify_vault_unsealed" {
-    module = module.vault_verify_unsealed
-    depends_on = [
-      step.create_vault_cluster,
-    ]
+    module     = module.vault_verify_unsealed
+    depends_on = [step.create_vault_cluster]
 
     providers = {
       enos = local.enos_provider[matrix.distro]
     }
 
     variables {
-      vault_instances  = step.create_vault_cluster.vault_instances
-      vault_root_token = step.create_vault_cluster.vault_root_token
+      vault_install_dir = local.vault_install_dir
+      vault_instances   = step.create_vault_cluster.vault_instances
+      vault_root_token  = step.create_vault_cluster.vault_root_token
     }
   }
 
   step "verify_raft_auto_join_voter" {
-    skip_step = matrix.backend != "raft"
-    module    = module.vault_verify_raft_auto_join_voter
-    depends_on = [
-      step.create_vault_cluster,
-    ]
+    skip_step  = matrix.backend != "raft"
+    module     = module.vault_verify_raft_auto_join_voter
+    depends_on = [step.create_vault_cluster]
+
+    providers = {
+      enos = local.enos_provider[matrix.distro]
+    }
+
+    variables {
+      vault_install_dir = local.vault_install_dir
+      vault_instances   = step.create_vault_cluster.vault_instances
+      vault_root_token  = step.create_vault_cluster.vault_root_token
+    }
+  }
+
+  step "verify_replication" {
+    skip_step  = matrix.artifact_type != "package"
+    module     = module.vault_verify_replication
+    depends_on = [step.create_vault_cluster]
+
+    providers = {
+      enos = local.enos_provider[matrix.distro]
+    }
+
+    variables {
+      vault_edition     = matrix.edition
+      vault_install_dir = local.vault_install_dir
+      vault_instances   = step.create_vault_cluster.vault_instances
+    }
+  }
+
+  step "verify_ui" {
+    skip_step  = matrix.artifact_type != "package"
```

Comment:
I'm also confused about skipping here. Seems to be a good idea to validate that all things we're testing are built with a UI